lib/utils/forkedblockchain.js

```diff
@@ -216,6 +218,7 @@ ForkedBlockchain.prototype.getFallbackBlock = function(numberOrHash, cb) {
     block.header.gasUsed = utils.toBuffer(json.gasUsed);
     block.header.timestamp = utils.toBuffer(json.timestamp);
     block.header.extraData = utils.toBuffer(json.extraData);
+    block.uncleHeaders = json.uncles;
```

Comment:
I wonder if we should guard against `json.uncles` not being an array, or not being an array with data formatted as we'd expect?

Reply:
That particular validation logic may not be reachable. Not sure if I can think of a scenario where uncles are malformed and we don't have more fundamental issues.